prime-router/src/test/kotlin/history/SubmissionHistoryTests.kt

```diff
@@ -692,7 +692,6 @@ class SubmissionHistoryTests {
         }
     }
 
-    // TODO: remove after route queue empty (see https://github.com/CDCgov/prime-reportstream/issues/15039)
```

Comment:
Looks like it was maybe intentional to keep this test?  I would maybe leave a comment indicating that it exists to test historical purposes?